ballerina-tests/tests/48_bidi_timestamp_service.bal

```diff
@@ -34,20 +34,20 @@ service "BidiStreamingTimestampService" on ep48 {
     }
 
     remote function bidiStreamingGreetBoth(BidiStreamingTimestampServiceBiDiGreetingCaller caller,
-     stream<BiDiGreeting, grpc:Error?> clientStream) returns error? {
-        check clientStream.forEach(function(BiDiGreeting value) {
+    stream<BiDiGreeting, grpc:Error?> clientStream) {
+        checkpanic clientStream.forEach(function(BiDiGreeting value) {
```

Comment:
```suggestion
        check clientStream.forEach(function(BiDiGreeting value) {
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/c32ef7e98e2bfaa4df966a96025d16d755dfb1cd